.env

```
@@ -42,3 +42,5 @@ SESSION_SECRET=secret
 ## Using `env` → POSTGRES_USER=${HASH_PG_USER} does not give access to values in this file.
 POSTGRES_USER=${HASH_PG_USER}
 POSTGRES_PASSWORD=${HASH_PG_PASSWORD}
+
+LOG_LEVEL=info
```

```suggestion
LOG_LEVEL=info
```

(EOL plz 🙏)

You can configure automatic EOL insertion in your editor. It reduces future diffs when we add more lines at the end of the file

---

Sure, have done so here: https://github.com/hashintel/hash/pull/322/commits/5b992de9ac707327da41f26bbcafbaf7c9f324ec